src/pages/Schedule.tsx

```diff
@@ -59,14 +53,19 @@ export const Schedule = () => {
       justifyContent="center"
       flexDirection="column"
     >
-      <Container mb={32} maxW="container.xlg">
+      <Container mb={32} maxW="container.xl">
         <Heading mb={6}>View Schedule</Heading>
         <Flex alignItems="center" justifyContent="space-between" mb={5}>
           <Select
             id="select"
             w="160px"
-            value={selectValue}
-            onChange={(e) => setSelectValue(e.target.value)}
+            value={viewState}
+            mb={5}
```

Comment:
```suggestion
```
This line should have been left out when you did the merge conflict resolution

Reply:
The margin below the select makes the alignment weird here
![image](https://user-images.githubusercontent.com/32287958/172073775-0495b0f7-7197-4dc0-b0c2-7504f29d748a.png)

Reply:
oops my bad, fixed it